localization/french/culture/wc_cultural_maa_innovations_l_french.yml

```
@@ -90,10 +90,10 @@
  innovation_ursa:0 "Ursas"
  innovation_ursa_desc:0 "Les Ursas sont les plus grands, les plus puissants et les plus féroces des furbolgs. Ils dominent les humains et les orcs et peuvent briser le cou des elfes d'un seul coup de patte. Ils sont immensément forts et résistants, et se concentrent sur l'amélioration de leur comportement naturel d'ours."
 
- innovation_warbrave:0 "Braves-guerriers"
+ innovation_warbrave:0 "Braves-Guerriers"
  innovation_warbrave_desc:0 "Devenir un valeureux tauren implique de surmonter plusieurs épreuves pour tester l'esprit, le corps et l'âme. La nature exacte des rites a changé avec le temps, mais la signification qui les motive est restée la même."
 
- innovation_war_dinosaur:0 "Dinosaures de guerre"
+ innovation_war_dinosaur:0 "Dinosaures de Guerre"
```

```suggestion
 innovation_war_dinosaur:0 "Dinosaures de guerre"
```
Je pense que la trad officielle a tendance a mettre une minuscule au second mot dans les noms comme ça.

---

Idem ici, après avoir checké en jeu je trouve ça plus lisible/"cool" avec une majuscule pour les troupes